website/static/js/jquery.homeycombs.js

```diff
@@ -0,0 +1,145 @@
+(function($) {

```

Comment:
Also, is there a reason you prefer to host the jquery files directly in the project instead of using a CDN?